parlai/agents/transformer/modules.py

```diff
@@ -288,6 +322,10 @@ def __init__(
         else:
             nn.init.normal_(self.position_embeddings.weight, 0, embedding_size ** -0.5)
 
+        #embedding normalization
```

Comment:
lint error here. You can run 
`flake8 path/to/my/dir` in ParlAI to see all of them.

Reply:
Or `bash tests/lint_changed.sh` to run the exact thing that CI runs